README.md

```diff
@@ -3,6 +3,7 @@
 An ESLint plugin for linting ESLint plugins. Rules written in CJS, ESM, and TypeScript are all supported.
 
 <!-- vscode-markdown-toc -->
+
```

Comment:
[Question] I kept having to undo Prettier changes to the README.md 😅 . Would you mind a separate issue around having it eslint-doc-generator and Prettier not come into conflict?

Reply:
Sure! https://github.com/bmish/eslint-doc-generator#prettier

Reply:
#434 -> #435 🚀